guevaraf-buenosaires.md

```diff
@@ -0,0 +1,7 @@
+## Buenos Aires, Argentina
+
+### Places to eat
+- [ ] Parrilla El 22
```

Comment:
Any chance you can tell them that I can't use gitter because the network doesn't allow chat!  I am sure there are others